hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/BlockID.java

```diff
@@ -65,6 +76,15 @@ public void setBlockCommitSequenceId(long blockCommitSequenceId) {
     this.blockCommitSequenceId = blockCommitSequenceId;
   }
 
+  // Can return a null value in case it is not set.
+  public Integer getReplicaIndex() {
+    return replicaIndex;
+  }
+
+  public void setReplicaIndex(Integer replicaIndex) {
+    this.replicaIndex = replicaIndex;
+  }
```

Comment:
This is unused.  If removed, `replicaIndex` member can be `final`.

Reply:
done